src/client/src/apps/MainRoute/widgets/analytics/components/analyticsLineChart/AnalyticsLineChart.tsx

```diff
@@ -115,6 +119,7 @@ const LineCharts: React.FC<LineChartProps> = React.memo(props => {
             padding={{ top: 0, bottom: 0 }}
             axisLine={false}
             tickFormatter={tickFormatYAxis}
+            domain={[dataMin, dataMax]}
```

Comment:
Nice one! Have you tried `domain={['dataMin', 'dataMax']}`, instead of calculating min/max yourself?